builtin/credential/cert/path_certs.go

```diff
@@ -47,36 +54,57 @@ Must be x509 PEM encoded.`,
 At least one must exist in either the Common Name or SANs. Supports globbing.  
 This parameter is deprecated, please use allowed_common_names, allowed_dns_sans, 
 allowed_email_sans, allowed_uri_sans.`,
+				DisplayAttrs: &framework.DisplayAttributes{
+					Group: "Constraints",
+				},
 			},
 
 			"allowed_common_names": &framework.FieldSchema{
 				Type: framework.TypeCommaStringSlice,
 				Description: `A comma-separated list of names.
 At least one must exist in the Common Name. Supports globbing.`,
+				DisplayAttrs: &framework.DisplayAttributes{
+					Group: "Constraints",
+				},
 			},
 
 			"allowed_dns_sans": &framework.FieldSchema{
 				Type: framework.TypeCommaStringSlice,
 				Description: `A comma-separated list of DNS names.
 At least one must exist in the SANs. Supports globbing.`,
+				DisplayAttrs: &framework.DisplayAttributes{
+					Name: "Allowed DNS SANs",
+					Group: "Constraints",
+				},
 			},
 
 			"allowed_email_sans": &framework.FieldSchema{
 				Type: framework.TypeCommaStringSlice,
 				Description: `A comma-separated list of Email Addresses.
 At least one must exist in the SANs. Supports globbing.`,
+				DisplayAttrs: &framework.DisplayAttributes{
+					Name: "Allowed Email SANs",
+					Group: "Constraints",
+				},
 			},
 
 			"allowed_uri_sans": &framework.FieldSchema{
 				Type: framework.TypeCommaStringSlice,
 				Description: `A comma-separated list of URIs.
 At least one must exist in the SANs. Supports globbing.`,
+				DisplayAttrs: &framework.DisplayAttributes{
+					Name: "Allowed URI SANs",
+					Group: "Constraints",
```

Comment:
the `Group` specifies which field group the field should belong to. for example:

![image](https://user-images.githubusercontent.com/903288/88088168-da5a8e00-cb3e-11ea-88ca-15df691d4374.png)